tests/hypothesis/test_base.py

```diff
@@ -43,8 +43,8 @@ async def test_can_use_fixture_provided_event_loop(event_loop, n):
     await semaphore.acquire()
 
 
-def test_async_auto_marked(pytester):
-    pytester.makepyfile(
+def test_async_auto_marked(testdir):
+    testdir.makepyfile(
```

Comment:
Required for pytest 6.1 as I see. Not a big problem though, the used API is the same.